verification.py

```diff
@@ -144,14 +145,19 @@ def check_slim_version(self):
             if version_list[i].lower() == 'version':
                 version_str = version_list[i+1]
                 break
-        version = float(version_str.strip(' ,'))
+        version = float(version_str.strip(' ,')[0:3])
         assert version >= min_version, "Require SLiM >= 3.1!"
 
     def get_ms_seeds(self):
         max_seed = 2**16
         seeds = [random.randint(1, max_seed) for j in range(3)]
         return ["-seed"] + list(map(str, seeds))
 
+    def get_discoal_seeds(self):
```

Comment:
Might as well just call ``get_ms_seeds``? If they're the same form then there's no point in this extra method.

Reply:
difference is in the flag we provide at the command line. `discoal` uses `-d` vs `ms`'s `-seed`

Reply:
Ah, didn't spot that! Ignore the above